binding/Binding/SKFont.cs

```diff
@@ -264,46 +264,46 @@ internal int CountGlyphs (void* text, int length, SKTextEncoding encoding)
 
 		// MeasureText
 
-		public float MeasureText (string text, SKPaint paint = null) =>
+		internal float MeasureText (string text, SKPaint paint = null) =>
 			MeasureText (text.AsSpan (), paint);
 
-		public float MeasureText (ReadOnlySpan<char> text, SKPaint paint = null)
+		internal float MeasureText (ReadOnlySpan<char> text, SKPaint paint = null)
 		{
 			fixed (void* t = text) {
 				return MeasureText (t, text.Length * 2, SKTextEncoding.Utf16, null, paint);
 			}
 		}
 
-		public float MeasureText (ReadOnlySpan<byte> text, SKTextEncoding encoding, SKPaint paint = null)
+		internal float MeasureText (ReadOnlySpan<byte> text, SKTextEncoding encoding, SKPaint paint = null)
 		{
 			fixed (void* t = text) {
 				return MeasureText (t, text.Length, encoding, null, paint);
 			}
 		}
 
-		public float MeasureText (IntPtr text, int length, SKTextEncoding encoding, SKPaint paint = null) =>
+		internal float MeasureText (IntPtr text, int length, SKTextEncoding encoding, SKPaint paint = null) =>
```

Comment:
I think there should be overloads that take a span of glyph indices. `ReadOnlySpan<ushort> glyphs`

Reply:
So all methods that except `SKTextEncoding` will use a hardcoded `SKTextEncoding.GlyphId`

Reply:
Very interesting... seems the C++ does not have that. But, I can add that. The interesting thing is that the current `measureText` method first gets the glyphs and measures that...

Reply:
> So all methods that except `SKTextEncoding` will use a hardcoded `SKTextEncoding.GlyphId`

I see. Just had to check the code... Sometimes the void* is restricted to various UTF* types.

Reply:
If it is not supported out of the box then it is fine.

Reply:
I am going to add it as it is just a matter of passing the glyph IDs. Nice of them to support glyphs.